PWGGA/GammaConvBase/AliConvEventCuts.cxx

```diff
@@ -4238,6 +4259,13 @@ Bool_t AliConvEventCuts::IsTriggerSelected(AliVEvent *event, Bool_t isMC)
       if (firedTrigClass.Contains(fSpecialSubTriggerName.Data())) isSelected = 1;
     }
 
+    // select manually PbPb kINT7 | kCentral | kSemiCentral
+    if ( fIsHeavyIon == 1 && fSpecialTrigger == 3  && fTriggerSelectedManually) {
+      if (firedTrigClass.Contains("CENT")){
+        if ( firedTrigClass.Contains("CV0H7") || firedTrigClass.Contains("CMID7") ) isSelected = 1;
```

Comment:
@mikesas please check that this breaks nothing for LHC11h either.

Reply:
Same comment as above, fSpecialTrigger == 3 is not used for 2.76TeV PbPb.